src/pages/User/Learning/Repeat/Repeat.tsx

```diff
@@ -0,0 +1,41 @@
+import * as S from './StyleRepeat';
+import ButtonIcon from 'components/Shared/Buttons/ButtonIcon';
+import { TitleBig } from 'components/Shared/Atoms/Title';
+import { inputNameElement } from 'helpers/mixins';
+import fetchWithToken from 'API/api';
+import { useState, useEffect } from 'react';
+import RepeatWords from 'components/Repeat/RepeatWords/RepeatWords';
+import RepeatForm from 'components/Repeat/RepeatForm/RepeatForm';
+
+interface RepeatProps {}
+
+const Repeat: React.FC<RepeatProps> = () => {
+  const [daysRepeat, setDaysRepeat] = useState<number>(7);
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
```

Comment:
jeśli typ jes tak oczywisty jak tutaj to nie musisz go deklarować. gdy value = `7` to ts wie, ze daysRepeat będzie `number`
i tak samo jak jest `false` to wie, ze będzie `boolean`